hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/CallIdSequenceWithBackoff.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
I wanted to make a minimal possible change to deliver the feature. with the exception of pulling out the inner class. I agree there is a little reason for client to have its own interface for callId generation.